mecademic_robot_node/src/mecademic_robot_driver/mecademic_robot_driver.py

```diff
@@ -168,16 +168,16 @@ def __del__(self):
         """Deconstructor for the Mecademic Robot ROS driver
         Deactivates the robot and closes socket connection with the robot
         """
-        self.robot.Deactivate()
-        self.robot.Disconnect()
-        self.feedback.Disconnect()
+        self.robot.DeactivateRobot()
+        self.robot.disconnect()
+        self.feedback.disconnect()
 
 if __name__ == "__main__":
-    robot = RobotController('192.168.0.100')
-    feedback = RobotFeedback('192.168.0.100')
-    robot.Connect()
-    feedback.Connect()
-    robot.Activate()
-    robot.Home()
+    robot = RobotController('192.168.1.9')
+    feedback = RobotFeedback('192.168.1.9', "v8.1.6.141")
+    robot.connect()
+    feedback.connect()
+    robot.ActivateRobot()
+    robot.home()
     driver = MecademicRobot_Driver(robot, feedback)
     rospy.spin()
```

Comment:
Can you add an empty line at the end of the file?

Reply:
I did that in the PEP PR because this one contains only functional changes, no spelling/style improvements.